### PR TITLE
[EASY] update ethereum.token_erc20

### PR DESCRIPTION
**Pull request by GoodDataBadGuy**
# Thank you for contributing to Spellbook!

Thank you for taking the time to submit code in Spellbook. A few things to consider:

- If you are a first-time contributor, please sign the CLA by copy & pasting **_exactly_** what the bot mentions in PR comment
- Refer to [docs](#spellbook-contribution-docs) section below to answer questions
- Dune team will review submitted PRs as soon as possible

### To speed up your development process in PRs, keep these tips in mind:

- Each commit to your feature branch will rerun CI tests ([see example](https://github.com/duneanalytics/spellbook/actions/runs/8202519819/job/22433451880?pr=5519))
    - This includes *all* modified models on your branch
    - This includes *all* history of the data
- Two tips for faster development iteration:
    - Ensure dbt is installed locally (refer to main `readme`) and run `dbt compile`
        - This will output raw SQL in `target/` directory to copy/paste and run on Dune directly for initial query testing
    - Hardcode a `WHERE` filter for only ~7 days of history on large source tables, i.e. `ethereum.transactions`
        - This will speed up the CI tests and output results quicker -- whether that's an error or fully successful run
        - Once comfortable with small timeframe, remove filter and let full history run

### Incremental model setup
- Make sure your unique key columns are *exactly* the same in the model config block, schema yml file, and seed match columns (where applicable)
- There cannot be nulls in the unique key columns
    - Be sure to double check key columns are correct or `COALESCE()` as needed on key column(s), otherwise the tests may fail on duplicates

### 🪄 Use the built CI tables for testing 🪄

Once CI completes, you can query the CI tables and errors in dune when it finishes running.
- For example:
    - In the `run initial models` and `test initial models`, there will be a schema that looks like this: `test_schema.git_dunesql_4da8bae_sudoswap_v2_base_pool_creations`
    - This can be temporarily queried in Dune for ~24 hours

Leverage these tables to perform QA testing on Dune query editor -- or even full test dashboards!

## Spellbook contribution docs

The [docs](docs) directory has been implemented to answer as many questions as possible. Please take the time to reference each `.md` file within this directory to understand how to efficiently contribute & why the repo is designed as it is 🪄

Example questions to be answered:

- What does each property in the [model config block](/docs/models/model_config_block.md) mean?
- What is the [CI test](/docs/ci_test/ci_test_overview.md) attached to PRs and how can I best utilize it?
- What are the Spellbook [best practices](/docs/general/best_practices.md)?

Please navigate through the [docs](/docs) directory to find as much info as you can.

**Note**: happy to take PRs to improve the docs, let us know 🤝

### CODE REVIEW

File: tokens/models/tokens/ethereum/tokens_ethereum_erc20.sql [hunk 1]
```diff
@@ -40148,5 +40148,9 @@ FROM (VALUES
         ,(0x096deae23d5e58f47c6ffc0b48d015756a2b21d3, 'FC', 18)
         ,(0xbe7458bc543cf2df43ac109d2f713dffe6417aa4, 'ATH', 9)
         ,(0x2048e0d048224381cac3ea06012ced4a6f122d32, '$BANDS', 18)
-        ,(0x695506f1583e678f9007dd47e86b5e850dbde0d4, 'PorkX', 9)      
+        ,(0x695506f1583e678f9007dd47e86b5e850dbde0d4, 'PorkX', 9)
+        ,(0x89ac16b14111a5f85d5532e30700e972ff9a77a9, 'xRWA', 18)
```
Comment (GoodDataBadGuy) [inline tokens/models/tokens/ethereum/tokens_ethereum_erc20.sql:40152]: https://etherscan.io/address/0x89ac16b14111a5f85d5532e30700e972ff9a77a9

File: tokens/models/tokens/ethereum/tokens_ethereum_erc20.sql [hunk 2]
```diff
@@ -40148,5 +40148,9 @@ FROM (VALUES
         ,(0x096deae23d5e58f47c6ffc0b48d015756a2b21d3, 'FC', 18)
         ,(0xbe7458bc543cf2df43ac109d2f713dffe6417aa4, 'ATH', 9)
         ,(0x2048e0d048224381cac3ea06012ced4a6f122d32, '$BANDS', 18)
-        ,(0x695506f1583e678f9007dd47e86b5e850dbde0d4, 'PorkX', 9)      
+        ,(0x695506f1583e678f9007dd47e86b5e850dbde0d4, 'PorkX', 9)
+        ,(0x89ac16b14111a5f85d5532e30700e972ff9a77a9, 'xRWA', 18)
+        ,(0x8b9b95292f890df47fff5ac9cbe93d5fc242bd51, 'BEFI', 18)
```
Comment (GoodDataBadGuy) [inline tokens/models/tokens/ethereum/tokens_ethereum_erc20.sql:40153]: https://etherscan.io/address/0x8b9b95292f890df47fff5ac9cbe93d5fc242bd51

File: tokens/models/tokens/ethereum/tokens_ethereum_erc20.sql [hunk 3]
```diff
@@ -40148,5 +40148,9 @@ FROM (VALUES
         ,(0x096deae23d5e58f47c6ffc0b48d015756a2b21d3, 'FC', 18)
         ,(0xbe7458bc543cf2df43ac109d2f713dffe6417aa4, 'ATH', 9)
         ,(0x2048e0d048224381cac3ea06012ced4a6f122d32, '$BANDS', 18)
-        ,(0x695506f1583e678f9007dd47e86b5e850dbde0d4, 'PorkX', 9)      
+        ,(0x695506f1583e678f9007dd47e86b5e850dbde0d4, 'PorkX', 9)
+        ,(0x89ac16b14111a5f85d5532e30700e972ff9a77a9, 'xRWA', 18)
+        ,(0x8b9b95292f890df47fff5ac9cbe93d5fc242bd51, 'BEFI', 18)
+        ,(0x8e4d27d772099e18900caf5dcc36cb612dcee886, 'ARCX', 18)
```
Comment (GoodDataBadGuy) [inline tokens/models/tokens/ethereum/tokens_ethereum_erc20.sql:40154]: https://etherscan.io/address/0x8e4d27d772099e18900caf5dcc36cb612dcee886

File: tokens/models/tokens/ethereum/tokens_ethereum_erc20.sql [hunk 4]
```diff
@@ -40148,5 +40148,9 @@ FROM (VALUES
         ,(0x096deae23d5e58f47c6ffc0b48d015756a2b21d3, 'FC', 18)
         ,(0xbe7458bc543cf2df43ac109d2f713dffe6417aa4, 'ATH', 9)
         ,(0x2048e0d048224381cac3ea06012ced4a6f122d32, '$BANDS', 18)
-        ,(0x695506f1583e678f9007dd47e86b5e850dbde0d4, 'PorkX', 9)      
+        ,(0x695506f1583e678f9007dd47e86b5e850dbde0d4, 'PorkX', 9)
+        ,(0x89ac16b14111a5f85d5532e30700e972ff9a77a9, 'xRWA', 18)
+        ,(0x8b9b95292f890df47fff5ac9cbe93d5fc242bd51, 'BEFI', 18)
+        ,(0x8e4d27d772099e18900caf5dcc36cb612dcee886, 'ARCX', 18)
+        ,(0xadd353fb2e2c563383ff3272a500f3e7134dafe4, 'TUNA', 18)
```
Comment (GoodDataBadGuy) [inline tokens/models/tokens/ethereum/tokens_ethereum_erc20.sql:40155]: https://etherscan.io/address/0xadd353fb2e2c563383ff3272a500f3e7134dafe4

**Review by Hosuke — APPROVED** [review]
LGTM.✅
Thank you @GoodDataBadGuy

File: tokens/models/tokens/ethereum/tokens_ethereum_erc20.sql [hunk 5]
```diff
@@ -40148,5 +40148,10 @@ FROM (VALUES
         ,(0x096deae23d5e58f47c6ffc0b48d015756a2b21d3, 'FC', 18)
         ,(0xbe7458bc543cf2df43ac109d2f713dffe6417aa4, 'ATH', 9)
         ,(0x2048e0d048224381cac3ea06012ced4a6f122d32, '$BANDS', 18)
-        ,(0x695506f1583e678f9007dd47e86b5e850dbde0d4, 'PorkX', 9)      
+        ,(0x695506f1583e678f9007dd47e86b5e850dbde0d4, 'PorkX', 9)
+        ,(0x89ac16b14111a5f85d5532e30700e972ff9a77a9, 'xRWA', 18)
+        ,(0x8b9b95292f890df47fff5ac9cbe93d5fc242bd51, 'BEFI', 18)
+        ,(0x8e4d27d772099e18900caf5dcc36cb612dcee886, 'ARCX', 18)
+        ,(0xadd353fb2e2c563383ff3272a500f3e7134dafe4, 'TUNA', 18)
+        ,(0x1495bc9e44Af1F8BCB62278D2bEC4540cF0C05ea, 'DEAI', 18)
```
Comment (GoodDataBadGuy) [inline tokens/models/tokens/ethereum/tokens_ethereum_erc20.sql:40156]: https://etherscan.io/address/0x1495bc9e44Af1F8BCB62278D2bEC4540cF0C05ea

**Comment by GoodDataBadGuy** [comment]
apologise, did not realised got reviewed. this is end of my commits and will create a new one if any more changes. thanks Dune Team!

**Comment by Hosuke** [comment]
> apologise, did not realised got reviewed. this is end of my commits and will create a new one if any more changes. thanks Dune Team!

It's ok, no worries.

**Review by Hosuke — APPROVED** [review]

**Pull request merged**